config/locales/bp/ru.new_customer.yml

```diff
@@ -33,17 +43,25 @@ ru:
     Activity_0dhj8f6:
       label: Подтверждение
       group1: Данные абонента
-      homsOrderDataCustomerName: Имя
-      homsOrderDataCustomerSurname: Фамилия
-      homsOrderDataCustomerPhone: Телефон
-      homsOrderDataCustomerEmail: Email
+      homsOrderDataCustomerName:
+        label: Имя
+      homsOrderDataCustomerSurname:
+        label: Фамилия
+      homsOrderDataCustomerPhone:
+        label: Телефон
+      homsOrderDataCustomerEmail:
+        label: Email
       homsOrderDataInstallDate: Дата подключения
```

Comment:
It's actually confusing: for some types of fields yous should use `label`, for other types you shouldn't. Make it consistent, please.